src/screenshotbot/compare.lisp

```diff
@@ -310,19 +320,10 @@
                 collect
                 <div class= "image-comparison-wrapper" >
                 <h3>,(screenshot-name before)</h3>
```

Comment:
comment for L318. In the (prepare-image-comparison ...) just add a `:size :half-page` as a parameter. That should fix the issue I mentioned.
(I added code to support image sizes for comparisons in https://github.com/screenshotbot/screenshotbot-oss/commit/931b8eda972fb399cbea24f1456454be69578657 and https://github.com/screenshotbot/screenshotbot-oss/commit/ad7f991c2e67446a7eb704a3445a94ef8bc7db84)

Reply:
Or may I change the other 2 images to `full-page` ?